File: Java/Easy/4168-mirror-distance-of-an-integer/mirror-distance-of-an-integer.java
class Solution {
    public int mirrorDistance(int n) {
        int rev=0,num=n,diff;
        while(num!=0){
            rev=rev*10+num%10; //% 10 extracts digits from right to left,* 10 pushes existing digits left
            num=num/10;
        }
        diff=n-rev;
        if(diff<0)
        diff=-diff;
        return diff;
    }
}

File: Python/Easy/4168-mirror-distance-of-an-integer/mirror-distance-of-an-integer.py
class Solution:
    def mirrorDistance(self, n: int) -> int:
        num=n
        rev=0
        ans=0
        while(num>0):
            rev=rev*10+num%10
            num//=10
        ans=rev-n
        if(ans<0):
            ans=-ans
        return ans
        